app/src/main/java/org/gdg/frisbee/android/activity/SettingsActivity.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 The GDG Frisbee Project
+ * Copyright 2013-2015 The GDG Frisbee Project

```

Comment:
:+1:

On February 8, 2015 9:44:21 PM CET, "Friedger Müffke" <notifications@github.com> wrote:
>> @@ -1,5 +1,5 @@
>>  /*
>> - * Copyright 2013 The GDG Frisbee Project
>> + * Copyright 2013-2015 The GDG Frisbee Project
>
>We should remove the copyright per file to avoid these changes in the
>future. The license in the root folder is usually accepted as
>sufficient and valid for all files in the project.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gdg-x/frisbee/pull/180/files#r24302151

--
Sent from mobile. Please excuse brevity and any typos.